libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -179,6 +180,66 @@ bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
 	return false;
 }
 
+// Preliminary implementation that doesn't take care of many things like
+// name collisions etc. Mainly used for test cases.
+bool IRGeneratorForStatements::visit(InlineAssembly const& _inlineAsm)
+{
+	auto const& loc = _inlineAsm.location();
+
+	solAssert(loc.start != 1 && loc.end != -1, "Expected source location");
+
+	vector<string> params, retParams, args;
+	vector<map<string, string>> assignVars;
+	// Prepare parameters and arguments
+	for (auto const& pair: _inlineAsm.annotation().externalReferences)
+	{
+		auto varDecl = dynamic_cast<VariableDeclaration const*>(pair.second.declaration);
+		solUnimplementedAssert(!!varDecl, "");
```

Comment:
Why `!!`?

Reply:
I think I copied that part from somewhere that used it.. I'll remove it